src/core/database/database.service.ts

```diff
@@ -580,7 +584,7 @@ export class DatabaseService {
       })
       .with('distinct(node) as node')
       //Mark baseNode labels and id deleted
-      .call(setBaseNodeLabelsAndIdDeleted, baseNodeLabels)
+      .call(setBaseNodeLabelsAndIdDeleted, nodeLabelsResult!.nodeLabels)
```

Comment:
I don't think we need change the ID, only the labels. I believe this can be done in one query as well. Perhaps this whole function needs to be rewritten.